app/uploaders/asset_uploader.rb

```diff
@@ -88,6 +88,22 @@ class AssetUploader < Kithe::AssetUploader
   end
 
 
+  # For FLAC originals, we create a mono m4a derivative.
+  # Typically this deriv is only 5% of the size of the original FLAC,
+  # while still fine for listening, at least for for oral histories.
+  # (These settings would not be appropriate for audio other than recorded speech.)
+  #
+  # See also the settings at app/services/combined_audio_derivative_creator.rb
+  # which are identical (similar use case).
+  Attacher.define_derivative('m4a', content_type: "audio") do |original_file, attacher:|
+    # Both audio/flac or audio/x-flac seem to be valid, so let's check for either.
+    if attacher.file.content_type.match /flac$/
```

Comment:
Why not just `== "audio/flac"? ` instead of `.match /flac$/`?

Reply:
Also may need some `&.` to avoid crashing if file.content_type is missing?

Reply:
Ah right that comment is right there. OK but how about `attacher.file.content_type.in?(["audio/flac", "audio/x-flac"])` :)

Reply:
Yeah good call on the `&.` I'll take the whole line.